tests/functional/test_libraries.py

```diff
@@ -8,15 +8,17 @@
 #-----------------------------------------------------------------------------
 
 
+# Library imports
+# ---------------
 import os
 import pytest
+import shutil
 
+# Local imports
+# -------------
 from PyInstaller.compat import is_win
 from PyInstaller.utils.tests import importorskip, xfail_py2
-
-
-# Directory with data for some tests.
-_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
+from tests.functional.conftest import DATA_DIR
```

Comment:
@codewarrior0, thanks so much for pointing this out. In my tree, there was still a tests/__init__.pyc left over from earlier executions, so I completely overlooked this in my testing. Ouch. I'll probably go for a fixture to solve this.